.env.sample

```diff
@@ -1,22 +1,26 @@
 NODE_ENV=development
 PORT=3004
 #HOSTNAME for AuthService
-HOST_HOSTNAME=
+HOST_HOSTNAME=localhost
 
+## Database settings
 DB_USERNAME=
 DB_PASSWORD=
 MYSQL_ROOT_PASSWORD=
 DB_PORT=3307 # This values maps mysql docker container to your local machine, if you already have something listening on port 3307 you will need to update this value to any available port.
 DB_NAME=authservice
-DB_HOSTNAME=auth_mysql_db
+DB_HOSTNAME=auth_mysql_db # If running on local machine change to localhost
 
-#Auth service UI Port
+## Auth service UI Port
 WEB_PORT=3003
-#SSL Auth service UI Port
+
+## SSL Auth service UI Port
```

Comment:
I thought we discussed to try the SSL_ENABLED=true/false and dynamically change the nginx config accordingly. Can we do that for both AuthService and Tombolo?

Reply:
Using IF  directive is not encouraged - https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/
Nginx docs say it cause unpredictable behaviour, including potential SIGSEGV and is  a good idea to avoid it if possible.
The if directive did not work for us anyway